### PR TITLE
Automation: Can perform basic commands on custom resources using workspace URL

### PR DESCRIPTION
**Pull request by zxiao-redhat**
This is an automation for test case (OCP-59389) that tests the basic commands (get/delete) on custom resources created on remote workspaces using workspace URL.

Test:
```
passed: (39.2s) 2023-01-30T05:02:27 "[area/workspaces] Author:zxiao-Low-[API] Can perform basic commands on custom resources using workspace URL [Suite:kcp/smoke/parallel]"
```

### CODE REVIEW

File: test/extended/workspacetype/workspace.go [hunk 1]
```diff
@@ -115,4 +115,36 @@ var _ = g.Describe("[area/workspaces]", func() {
...
+		o.Expect(output).Should(o.ContainSubstring(rsName))
+
+		g.By("# Test delete command on the custom resource")
+		err = k.WithoutWorkSpaceServer().Run("delete").Args(resourceName, "--server="+serverURL).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Run get command on the custom resource")
+		output, err = k.WithoutWorkSpaceServer().Run("get").Args(resourceName, "--server="+serverURL).Output()
+		o.Expect(err).To(o.HaveOccurred())
+		o.Expect(output).Should(o.ContainSubstring("not found"))
```
Comment (Phaow) [inline test/extended/workspacetype/workspace.go]:
```suggestion
                o.Eventually(func() string {
			output, _ := k.WithoutWorkSpaceServer().Run("get").Args(resourceName, "--server="+serverURL).Output()
			return output
		}, 60*time.Second, 5*time.Second).Should(o.ContainSubstring("not found"))
```
Hi @zxiao-redhat , We'd better use asynchronous assertion for the step, as the delete action may need a while， like some resource may keep terminating status for a while after the terminating it deleted successfully.

Comment (zxiao-redhat) [reply test/extended/workspacetype/workspace.go]: Hi @Phaow, thank you for your suggestions. I updated the PR. PTAL.

File: test/extended/workspacetype/workspace.go [hunk 2]
```diff
@@ -115,4 +115,40 @@ var _ = g.Describe("[area/workspaces]", func() {
...
+
+		g.By("# Test delete command on the custom resource")
+		err = k.WithoutWorkSpaceServer().Run("delete").Args(resourceName, "--server="+serverURL).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Run get command on the custom resource")
+		o.Eventually(func() string {
+			output, err = k.WithoutWorkSpaceServer().Run("get").Args(resourceName, "--server="+serverURL).Output()
+			if err != nil {
+				return "get command failed, try again"
```
Comment (Phaow) [inline test/extended/workspacetype/workspace.go:151]: if we want to handdle the error info, we'd better also print out it.
```suggestion
				e2e.Logf("Get %s failed of: %v", resourceName,  err)
				return "get command failed, try again"
```

**Comment by Phaow** [comment]
LGTM

**Pull request merged**